src/content/advanced/gitops/index.md

```diff
@@ -125,367 +123,193 @@ flux-multi-tenancy:
 
-```
+First thing's first - create a bare-bones repository where we will store the resources that will be deployed. In this repository we need to create the structure described in the last section, and copy the bases for workload clusters and other apps in the Giant Swarm catalog. Follow these steps:
+  
+1. Copy bases folder from [giantswarm/gitops-template](https://github.com/giantswarm/gitops-template)
 
-or Flux CLI - `flux`:
+2. Init the repository in order to create the first part of the structure and some git hooks. This will create a folder `management-clusters`. Use the command:
 
 ```nohighlight
- flux get sources git flux-demo
-NAME            READY   MESSAGE                                                         REVISION                                        SUSPENDED
-flux-demo       True    Fetched revision: main/74f8d19cc2ac9bee6f45660236344a054c63b71f main/74f8d19cc2ac9bee6f45660236344a054c63b71f   False
+kubectl gs gitops init
 ```
 
-### Managing organizations
+> Note: To learn more about Giant Swarm's kubectl plugin, visit [kubectl-gs documentation]({{< relref "/ui-api/kubectl-gs/" >}}).
 
-The first Giant Swarm custom resource you can create is an [Organization]({{< relref "/ui-api/management-api/crd/organizations.security.giantswarm.io.md" >}}). We will use `Kustomization` resource to tell Flux where to find YAML files defining resources and where to install them.
+### Attach the repo to a management cluster
 
-File structure
+Now we create the structure for the management clusters we are going to deploy to. Run this command for every management cluster.
 
 ```nohighlight
-.
-├── 02-organization.yaml    # contains Kustomization CR
-├── 02-organization
-│   ├── kustomization.yaml  # kustomize manifest
-│   └── resources.yaml      # contains Organization CR
+kubectl gs gitops add management-cluster --name MC_NAME --repository-name MC_NAME-git-repo --gen-master-key
 ```
 
-First, let's create the kustomize directory:
+This commands creates the folder for the management cluster, the `Kustomization` file for the MC, `organizations` folder and `secrets` folder, already populated with an encoding key.
 
-```nohighlight
-mkdir 02-organization
-```
+Then, we create `GitRepository` resource that points to the repository. We will need a resource for every management cluster, so we filter the files to be retrieved.
 
-```nohighlight
-echo -e 'resources:\n- resources.yaml' > 02-organization/kustomization.yaml
-```
+First, we need to crate a secret (we will refer to it as GIT_CREDENTIALS_SECRET, please, replace with the chosen name) with the credentials to access the repository. This will allow Flux to download the repo files in order to apply all resources. If your repo is public you don't need the secret and can safely delete the parameter `secret-ref` of the next command.
+This process is different depending on the git platform used (ssh-keys, token, user/password). Check documentation of every provider to create the secret.
```

Comment:
```suggestion
First, we need to create a secret (we will refer to it as `GIT_CREDENTIALS_SECRET`, please, replace it with the chosen name) with the credentials to access the repository. This will allow Flux to download the repo files in order to apply all resources. If your repo is public you don't need the secret and can safely delete the parameter `secret-ref` of the next command.

This process is different depending on the git platform used (ssh-keys, token, user/password). Check the documentation of every provider to create the secret.
```